app/assets/stylesheets/shared/interactive.scss

```diff
@@ -1,4 +1,4 @@
 .hidden-irrelevant, .hidden-deleted, .hidden-closed {
-  display: none;
+  display: none !important;
```

Comment:
Yeah, definitely. But the idea is that these classes guarantee that if any of them is applied, it will not display. I believe there was a bootstrap animation that applied inline styles, and you'll never be able to override those without `!important`, which is frustrating cause it means you don't get different hiding contexts.